File: WebAdminApp/Models/SoccerEventDto.cs
using System;
using Core.Enums;

namespace WebAdminApp.Models
{
    public class SoccerEventDto
    {
        public Guid Id { get; set; }
        public SoccerEventType SoccerEventType { get; set; }
        public SoccerPlayerDto SoccerPlayer { get; set; }
    }
}

File: WebEmployeeApp/ClientApp/src/app/models/SoccerEventDto.ts
import { SoccerEventType } from "./enums/SoccerEventType";
import { SoccerPlayerDto } from "./SoccerPlayerDto";

export interface SoccerEventDto {
  id: string;
  soccerGameId: string;
  soccerTeamId: string;
  soccerPlayer: SoccerPlayerDto;
  soccerEventType: SoccerEventType;
}

export interface CreateSoccerEventDto {
  soccerGameId: string;
  soccerTeamId: string;
  soccerPlayerId: string;
  soccerEventType: SoccerEventType;
}
